insecure/wintermute/attackOrchestrator.go

```diff
@@ -90,18 +92,37 @@ func (o *Orchestrator) HandleEventFromCorruptedNode(event *insecure.EgressEvent)
 
 	default:
 		// Any other event is just passed through the network as it is.
-		err := o.network.Send(event)
+		err := o.network.SendEgress(event)
 
 		if err != nil {
-			return fmt.Errorf("could not send rpc on channel: %w", err)
+			return fmt.Errorf("could not send rpc egress event on channel: %w", err)
 		}
 		o.logger.Debug().
-			Hex("corrupted_node_id", logging.ID(event.CorruptedNodeId)).
-			Str("channel_id", string(event.Channel)).
+			Hex("corrupt_origin_id", logging.ID(event.CorruptOriginId)).
+			Str("channel", string(event.Channel)).
 			Str("protocol", event.Protocol.String()).
 			Str("type", fmt.Sprintf("%T", event)).
-			Msg("miscellaneous event has passed through")
+			Msg("egress event has passed through")
+	}
+
+	return nil
+}
+
+// HandleIngressEvent implements logic of processing the incoming (ingress) events to a corrupt node.
+// Wintermute orchestrator doesn't corrupt ingress events, so it just passes them through.
+func (o *Orchestrator) HandleIngressEvent(event *insecure.IngressEvent) error {
+	err := o.network.SendIngress(event)
+
+	if err != nil {
+		return fmt.Errorf("could not send rpc ingress event on channel: %w", err)
 	}
+	o.logger.Debug().
+		Hex("origin_id", logging.ID(event.OriginID)).
+		Str("channel", string(event.Channel)).
+		Str("corrupt_target_id", fmt.Sprintf("%v", event.CorruptTargetID)).
+		Str("flow_protocol_event", fmt.Sprintf("%T", event.FlowProtocolEvent)).
+		Str("type", fmt.Sprintf("%T", event)).
+		Msg("ingress event has passed through")
```

Comment:
Here we only log on happy path. But it is more helpful to log both the happy and unhappy paths with the same context, i.e., `origin_id`, `channel`, etc.

Reply:
This was actually on my list of things to update in a future PR but got it done here by standardizing the way logging is done for both Wintermute and Dummy orchestrators.

https://github.com/onflow/flow-go/pull/2936/commits/62317e60f52c832168e6ac3826bdd50ebd0c8fa5